crates/biome_analyze/src/lib.rs

```diff
@@ -2,7 +2,6 @@
 
 use biome_console::markup;
 use biome_parser::AnyParse;
-use std::cmp::Ordering;
```

Comment:
I think you made a mistake and added a commit/change from the other PR

Reply:
sorry, my git broken, i fix this